lib/Bindings/Python/support.py

```diff
@@ -165,6 +165,19 @@ def attribute_to_var(attr):
   except ValueError:
     pass
 
+  from .dialects import om
+  try:
+    return attribute_to_var(om.ReferenceAttr(attr).inner_ref)
+  except ValueError:
+    pass
+
+  from .dialects import hw
```

Comment:
We normally don't do the imports within functions, but I think in this case we needed it from some reason. Since it's already kind of unusual, do you mind doing all the local imports on one line like `from .dialects import hw, om` before all the `try` statements like in `type_to_pytype`? Since it's a slightly weird thing, I think best to do it once at the top rather than throughout.